magefiles/go.mod

```diff
@@ -1,6 +1,8 @@
 module github.com/mesosphere/kommander-applications/magefiles
 
-go 1.20
+go 1.21
+
+toolchain go1.21.6
```

Comment:
This change seems to be applied automatically (by goland?) @mikolajb do we need this line of `toolchain go1.21.6`? I saw `kommander` repo doesnt have it so I just removed this for `appscenarios` package's go .mod